pkgs/tools/system/pciutils/default.nix

```diff
@@ -11,10 +12,18 @@ stdenv.mkDerivation rec {
     sha256 = "1ss0rnfsx8gvqjxaji4mvbhf9xyih4cadmgadbwwv8mnx1xvjh4x";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config hwdata ];
   buildInputs = [ zlib kmod which ] ++
     lib.optional stdenv.hostPlatform.isDarwin darwin.apple_sdk.frameworks.IOKit;
 
+  postPatch = ''
+    # Update database with newer version from hwdata. We don't want to depend
+    # at runtime on hwdata, however, because this would inflate the closure
+    # size of pciutils with too much stuff which pciutils doesn't need.
+    # See #112644 for discussion of alternative solutions.
+    cp --reflink=auto ${hwdata}/share/hwdata/pci.ids pci.ids
+  '';
+
```

Comment:
If it's done here, I don't think reflink really helps.  For it to make sense, I can imagine instead replacing `$out/share/pci.ids` during `postInstall` or something similar.

Reply:
Oh, you're right. I'll fix that in the imminent force-push!